cirq-core/cirq/contrib/qasm_import/_parser_test.py

```diff
@@ -233,6 +234,62 @@ def test_classical_control():
     ct.assert_same_circuits(parsed_qasm.circuit, expected_circuit)
     assert parsed_qasm.qregs == {'q': 2}
 
+    # Note this cannot *exactly* round-trip because the way QASM and Cirq handle measurements
+    # into classical registers is different. Cirq parses QASM classical registers into m_a_i for i
+    # in 0..bit_count. Thus the generated key has an extra "_0" at the end.
+    expected_generated_qasm = """// Generated from Cirq v0.15.0.dev
```

Comment:
should this use `cirq.__version__` so that it doesn't break when we bump versions?